engine/runtime/src/bench/java/org/enso/interpreter/bench/benchmarks/semantic/ListBenchmarks.java

```diff
@@ -51,8 +51,25 @@ public void initializeBenchmark(BenchmarkParams params) throws Exception {
       from Standard.Base.Data.List.List import Cons, Nil
       import Standard.Base.IO
 
+      type Lenivy
+          Nic
+          Hlava ~x ~xs
+
+          map self fn = case self of
+              Lenivy.Nic -> Lenivy.Nic
+              Lenivy.Hlava x xs -> Lenivy.Hlava (fn x) (xs.map fn)
+
       plus_one list = list.map (x -> x + 1)
 
+      leniva_suma list acc = case list of
```

Comment:
I like the names 😁

Reply:
There are two ways to say "lazy" in Czech. [líný](https://slovnik.seznam.cz/preklad/cesky_polsky/l%C3%ADn%C3%BD) and [lenivý](https://slovnik.seznam.cz/preklad/cesky_polsky/leniv%C3%BD). Selected the second one as it translates better to Polish.